src/crypto/aead.rs

```diff
@@ -50,8 +50,8 @@ mod ring {
     }
 
     impl From<[u8; ring::aead::NONCE_LEN]> for FrameNonceSequence {
-        fn from(nonce: [u8; ring::aead::NONCE_LEN]) -> Self {
-            Self { buffer: nonce }
+        fn from(buffer: [u8; ring::aead::NONCE_LEN]) -> Self {
+            Self { buffer }
```

Comment:
do we maybe want to stick with the name `nonce`? seems to be more descriptive

Reply:
But in the end the outer type is `FrameNonceSequence` so I think the context is clear.